src/App.jsx

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable linebreak-style */
```

Comment:
- Disabling the same rule for all files is bad, you should have used `.eslintrc`
- The idea of having eslint rules is to use them. You could have used the `.editorconfig`  file we provided to set your editor to use the right linebreak style